src/Input/Input.js

```diff
@@ -89,6 +129,22 @@ module.exports = kind(
 	},
 
 	/**
+	* @private
+	*/
+	components: [
+		{kind: TooltipDecorator, autoShow: false, components: [
+			{name: 'tooltip', kind: Tooltip, floating: false, position: 'below'}	// To Do : It's position has dependency with ENYO-2709.
```

Comment:
First, I changed position value from 'right top' to 'below'.
I guess, sideways tooltip is displayed on wrong position when I set floating to 'true'.
Like as I commented in Jira, I think this tooltip is not moved when scrolling up and down.